lib/Search/UI/SearchResultsController.php

```diff
@@ -99,7 +99,7 @@ public function display()
                 'prev' => $page > 1,
                 'next' => $total - $from > $size,
                 'page' => $page,
-                'last' => (int)($total / $size) + 1,
+                'last' => (int)($total / $size) + ($total%$size === 0 ? 0 : 1),
```

Comment:
This will throw an error if $size is 0

Example:
index.php?action=Search&search-query-string=Asia&search-query-size=0&search-query-from=0&search-engine=ElasticSearchEngine

Reply:
Yes, It's right, however `search-query-size` query parameter never can be 0. Also I added a commit to handle it in back-end too.